docs/ops/sequence/AUGRUSequence_10.md

```diff
@@ -0,0 +1,145 @@
+
+  * **Description**: *activations* specifies activation functions for gates, there are two gates, so two activation functions should be specified as a value for this attributes
+  * **Range of values**: any combination of *relu*, *sigmoid*, *tanh*
+  * **Type**: a list of strings
+  * **Default value**: *sigmoid,tanh*
+  * **Required**: *no*
+
+* *activations_alpha, activations_beta*
+
+  * **Description**: *activations_alpha, activations_beta* attributes of functions; applicability and meaning of these attributes depends on chosen activation functions
+  * **Range of values**: a list of floating-point numbers
+  * **Type**: `float[]`
+  * **Default value**: None
+  * **Required**: *no*
+
+* *clip*
+
+  * **Description**: *clip* specifies bound values *[-C, C]* for tensor clipping. Clipping is performed before activations.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Default value**: *infinity* that means that the clipping is not applied
+  * **Required**: *no*
+
+* *direction*
+
+  * **Description**: Specify if the RNN is forward, reverse, or bidirectional. If it is one of *forward* or *reverse* then `num_directions = 1`, if it is *bidirectional*, then `num_directions = 2`. This `num_directions` value specifies input/output shape requirements.
+  * **Range of values**: *forward*, *reverse*, *bidirectional*
+  * **Type**: `string`
+  * **Required**: *yes*
+
+* *linear_before_reset*
+
+  * **Description**: *linear_before_reset* flag denotes, if the output of hidden gate is multiplied by the reset gate before or after linear transformation. It behaves according to the *ONNX GRU* described in the [ONNX documentation](https://github.com/onnx/onnx/blob/master/docs/Operators.md#GRU).
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: False
+  * **Required**: *no*
+
+**Inputs**
+
+* **1**: `X` - 3D tensor of type *T1* `[batch_size, seq_length, input_size]`, input data. **Required.**
+
+* **2**: `initial_hidden_state` - 3D tensor of type *T1* `[batch_size, num_directions, hidden_size]`, input hidden state data. **Required.**
+
+* **3**: `sequence_lengths` - 1D tensor of type *T2* `[batch_size]`, specifies real sequence lengths for each batch element. **Required.**
+
+* **4**: `W` - 3D tensor of type *T1* `[num_directions, 3 * hidden_size, input_size]`, the weights for matrix multiplication, gate order: zrh. **Required.**
+
+* **5**: `R` - 3D tensor of type *T1* `[num_directions, 3 * hidden_size, hidden_size]`, the recurrence weights for matrix multiplication, gate order: zrh. **Required.**
+
+* **6**: `B` - 2D tensor of type *T*. If *linear_before_reset* is set to 1, then the shape is `[num_directions, 4 * hidden_size]` - the sum of biases for z and r gates (weights and recurrence weights), the biases for h gate are placed separately. Otherwise the shape is `[num_directions, 3 * hidden_size]`, the sum of biases (weights and recurrence weights). **Required.**
```

Comment:
Is B required or optional?

Reply:
**B** (bias) input is required, the input can be filled with `0` if not needed.
It is also marked as "Required" in [GRUSequence_5.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/GRUSequence_5.md) and other Sequence ops [RNNSequence_5.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/RNNSequence_5.md), [LSTMSequence_1.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/LSTMSequence_1.md)